### PR TITLE
cilium: Fix image revision

### PR DESCRIPTION
**Pull request by mrostecki**
Signed-off-by: Michal Rostecki <mrostecki@opensuse.org>

### CODE REVIEW

**Comment by pablochacin** [comment]
@evrardjp Your concerns are valid. This doesn't need to be backported.

cc @jordimassaguerpla

**Comment by pablochacin** [comment]
@mrostecki  the PR still fails with the same error:
```
11 12:06:08 0100164089227 kubelet[6802]: E0811 12:06:08.951910    6802 kubelet.go:2188] Container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: No CNI configuration file in /etc/cni/net.d/. Has your network provider started
```

File: internal/pkg/skuba/kubernetes/versions.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ var (
 				Tooling:           &ContainerImageTag{Name: "skuba-tooling", Tag: "0.1.0"},
 			},
 			AddonsVersion: AddonsVersion{
-				Cilium:        &AddonVersion{"1.7.6", 5},
+				Cilium:        &AddonVersion{"1.7.6", 2},
```
Comment (jenting) [inline internal/pkg/skuba/kubernetes/versions.go:105]:
```suggestion
				Cilium:        &AddonVersion{"1.7.6-rev2", 5},
```
?

**Comment by jordimassaguerpla** [comment]
I don't think this is needed.

This was good already in https://github.com/SUSE/skuba/pull/1303

The issue was that the cilium image was in the Containers:CR project but have not been released into the Containers subproject.

**Pull request closed without merging**